### PR TITLE
fix: 4950 price fltr on active k-blue

### LINKED ISSUE #4950: Price filter logic
image
- the currency option there is for the future. 

changes: 
- default state of the button is disabled
- when you input something that actually makes sense it changes color to k-accent, it could be only one parameter, or both, as long as it makes sense its ok. 
- When you click apply, the button is disabled again until you change the filter. We are not keeping the numbers in the input. Once you click apply the filter is applied and added as a breadcrumbs - I will make issue for this as well. 

### active state of the input
- Lets add k-blue stroke on the outside 
like this: 
image

### PR DESCRIPTION
**Pull request by maderi21**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #4950
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [x] I've tested it at </bsx/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
image

### CODE REVIEW

File: components/explore/SidebarFilter.vue [hunk 1]
```diff
@@ -137,6 +153,22 @@ const replaceUrl = (queryCondition: { [key: string]: any }) => {
...
 }
+
+const isValidFilter = (
+  min: number | undefined,
+  max: number | undefined
+): boolean => {
+  if (min && max) {
+    return max > min
+  }
+  return false
```
Comment (Jarsen136) [inline components/explore/SidebarFilter.vue:164]: what if `min` is zero? It would also return false

**Review by Jarsen136 — CHANGES_REQUESTED** [review]

**Comment by exezbcz** [comment]
hi! 

## Feedback
### Disabled button
- should not have drop shadow
image
- disabled state should not be white
- active state is pink (k-accent) also on the picture (light/dark mode) 
image

figma for this: 
https://www.figma.com/file/OwAEyE0QDPn3tHXZbyot1o/Explore-v0.9?node-id=0%3A1&t=Q0EhchN7FTbyJNxY-1

### Input
Next thing is the logic. Something is wrong there, can you have a look please?

**Comment by maderi21** [comment]
- @Jarsen136 @preschian fixed the logic
- @exezbcz fixed the styling

File: components/explore/SidebarFilter.vue [hunk 2]
```diff
@@ -137,6 +155,25 @@ const replaceUrl = (queryCondition: { [key: string]: any }) => {
...
+    max !== undefined &&
+    min !== '' &&
+    max !== '' &&
+    Number(max) > Number(min)
+  )
+}
+
+const inputFocused = ref(false)
+
+const isInputFocused = (): void => {
```
Comment (Jarsen136) [inline components/explore/SidebarFilter.vue:174]:
```suggestion
const toggleInputFocused = (): void => {
```

**Review by Jarsen136 — COMMENTED** [review]
I guess we do not have to input the 0 to `MIN` , if we would like to search something is cheaper than the MAX price. I would like the apply button to be avaliable.

image
The same case about MAX input
image


This is the filter on the Opensea.
image

File: components/explore/SidebarFilter.vue (hunk 2 above)
Comment (maderi21) [reply components/explore/SidebarFilter.vue]: done

**Comment by maderi21** [comment]
> I guess we do not have to input the 0 to `MIN` , if we would like to search something is cheaper than the MAX price. I would like the apply button to be avaliable.
> 
> image
> 
> The same case about MAX input
> image
> 
> This is the filter on the Opensea. image

added this option 👍

**Review by Jarsen136 — APPROVED** [review]
WFM

**Comment by yangwao** [comment]
pay 40 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $40 @ 33.23 USD/KSM ~ 1.204 $KSM
🧗 [JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa](https://kusama.subscan.io/account/JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa)
🔗 [0xa36f1a7111b44c56a147b741e48e69f51c22c49053cb03e980ac953f8385c48a](https://kusama.subscan.io/extrinsic/0xa36f1a7111b44c56a147b741e48e69f51c22c49053cb03e980ac953f8385c48a)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**